Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -26,12 +26,17 @@ namespace Microsoft.MixedReality.Toolkit
     /// The Profile can be swapped out at any time to meet the needs of your project.
     /// </summary>
     [DisallowMultipleComponent]
+    [ExecuteAlways]
```

Comment:
What is the consequence of setting MRTK class to ExecuteAlways? Why do we need this?

Reply:
This ensures that its OnEnable / OnDisable will be called when you're not in playmode, and I'm using those to detect and register instances.